data/hai/hai fleets.txt

```diff
@@ -874,6 +874,33 @@ fleet "Large Plundering Unfettered"
 		"Sea Scorpion (Pulse)"
 		"Shield Beetle (Pulse)"
 
+fleet "Unfettered Hai Space Raid"
+	government "Hai (Unfettered Raiders)"
+	names "hai"
+	cargo 1
+	personality
+		disables plunders coward harvests
+	variant
+		"Lightning Bug (Pulse)" 7
+	variant 2
+		"Shield Beetle" 2
+	variant 2
+		"Shield Beetle (Pulse)" 2
+	variant 3
+		"Sea Scorpion (Pulse)" 5
+	variant 2
+		"Sea Scorpion (Ion)" 5
+	# run, you fool! (bigger less likely ones to prevent easy farming)
+	variant 6
+		"Shield Beetle (Pulse)"
+		"Sea Scorpion (Pulse)"
+		"Lightning Bug (Pulse)" 2
+	variant 10
+		"Sea Scorpion (Shredder)"
+		"Sea Scorpion (Ionic)"
+		"Shield Beetle (Pulse)"
+		"Shield Beetle"
```

Comment:
Your numbers are reversed from what your comment says. Higher numbers are _more_ likely, not less.